Pre-commit fails if locale files are not loaded

First [reported](https://github.com/WordPress/openverse-frontend/issues/1584#issuecomment-1201316092) by @ritesh-pandey.

## Description
<!-- Concisely describe the bug. Compare your experience with what you expected to happen. -->
<!-- For example: "I clicked the 'submit' button and instead of seeing a thank you message, I saw a blank page." -->
Pre-commit hooks try to run tests and fail if the locale files do not exist. 

## Reproduction
<!-- Provide detailed steps to reproduce the bug. -->
1. Remove all untracked files inside the `src/locales/` folder, if you have any (in particular, delete the `src/locales/ar.json` file).
2. Try making some changes and committing them using `git commit -m "Commit message".
3. Commit fails with the following error:
 ```
test/playwright/utils/navigation.ts:15:25 - error TS2307: Cannot find module '~/locales/ar.json' or its corresponding type declarations.

15 import rtlMessages from '~/locales/ar.json'
                           ~~~~~~~~~~~~~~~~~~~


Found 1 error in test/playwright/utils/navigation.ts:15

 ELIFECYCLE  Command failed with exit code 2.
husky - pre-commit hook exited with code 1 (error)
 ```

## Additional context
<!-- Add any other context about the problem here; or delete the section entirely. -->
If you run `pnpm dev` or `pnpm i18n`, locale files are loaded, and pre-commit hooks succeed even without this change.

## Resolution
<!-- Replace the [ ] with [x] to check the box. -->
- [x] 🙋 I would be interested in resolving this bug.

--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -1,5 +1,9 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+if [ ! -f "$(dirname "$dir")"/src/locales/ar.json ]; then
+  echo "Loading locales."
+  pnpm i18n
+fi
 # Lint staged files
 pnpx lint-staged && pnpm types

